src/draw/DefaultDrawer.js

```diff
@@ -13,29 +13,15 @@ class DefaultDrawer {
   /**
    * Default constructor
    * @param {Object} [resources=null] - Object that contains resources.
+   * @param {Object} [events=null] - Object that contains event subjects.
    * @param {String} [rootId="root"] - Id of HTML element where we want to draw.
-   * @param {Object} [events] - Events list.
-   * @param {Function} [events.SelectEvent.next] - Function to emit selection event.
```

Comment:
Why do you remove the event documentation ?

Reply:
events was unused at the time, put it back in.